### PR TITLE
Adding stac-fastapi documentation, pvcs for stac-fastapi & minio and stac-fastapi secret changes

### PR DESCRIPTION
**Pull request by hectormachin**
…other fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,50 +2,91 @@
...
 The local development environment includes a module for deploying STAC-FastAPI, which provides an API that exposes a STAC catalog that is persisted in a backend Postgres database with its component STAC Collections and Items. More information is available [here](./modules/stac-fastapi/README.md).
 
 You can configure whether or not STAC-FastAPI is included in deployment by setting the `deploy_stacfastapi` flag in the [inputs](./inputs.tf) file.
+<br></br>
+
+### SWOOP
+
+The swoop module contains the SWOOP Bundle which will install the followin the following components into a kubernetes cluster:
+* [swoop-api](https://github.com/Element84/swoop) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_api` flag in the [inputs](./inputs.tf) file
+* [argo-workflows](https://github.com/argoproj/argo-workflows/) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_argo_workflows` flag in the [inputs](./inputs.tf) file
```
Comment (geomatician) [inline README.md:67]: whether or not Argo Workflows is included

File: README.md [hunk 2]
```diff
@@ -2,50 +2,91 @@
...
+* [swoop-caboose](https://github.com/Element84/swoop-go) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_caboose` flag in the [inputs](./inputs.tf) file
```
Comment (geomatician) [inline README.md:68]: "whether or not Swoop Caboose is included"

File: README.md [hunk 3]
```diff
@@ -2,50 +2,91 @@
...
+* [swoop-conductor](https://github.com/Element84/swoop-go) - configure whether or not SWOOP-API is included in deployment by setting the `deploy_swoop_conductor` flag in the [inputs](./inputs.tf) file
```
Comment (geomatician) [inline README.md:69]: "whether or not Swoop Conductor is included"

File: README.md (hunk 1 above)
Comment (hectormachin) [reply README.md]: fixed, thanks for catching that

File: README.md (hunk 3 above)
Comment (hectormachin) [reply README.md]: fixed, thanks for catching that

**Review by geomatician — APPROVED** [review]

**Pull request merged**